src/Sylius/Bundle/AddressingBundle/Resources/config/validation/Province.xml

```diff
@@ -34,7 +34,7 @@
             </constraint>
             <constraint name="Regex">
                 <option name="message">sylius.province.code.regex</option>
-                <option name="pattern">/[A-Z]{2}-[^W\d]{2,}/</option>
```

Comment:
What about provinces identified by three or more letters? Shall we allow for numbers in province codes?